AGENTS.md

```diff
@@ -26,31 +25,31 @@
 All tests must follow this strict structure:
 
 ```typescript
-describe('function-name', () => {
+describe("function-name", () => {
```

Comment:
It looks like this code in the markdown file got formatted with the defaults (double quote, semicolon) but not reverted when the rules got adjusted

Reply:
Oops, nice catch. My IDE actually formatted this on save with prettier 🤦 Updated